cql/fhir-cql-server/src/main/java/com/ibm/fhir/cql/engine/server/retrieve/ServerFHIRRetrieveProvider.java

```diff
@@ -95,12 +94,12 @@ private Resource executeQuery(String dataType, MultivaluedMap<String, String> qu
         try {
             if (queryParameters.containsKey("_id")) {
                 String id = queryParameters.getFirst("_id");
-                SingleResourceResult<?> result = resourceHelpers.doRead(dataType, id, true, false, null);
+                SingleResourceResult<?> result = resourceHelpers.doRead(dataType, id, true);
```

Comment:
FHIRResourceHelpers has constants for things like DO_VALIDATION. I think a boolean constant for THROW_EXC_ON_NULL would improve readability a bit. But not essential.

Reply:
I checked all the callers of the simpler 3-arg `doRead` and I found that they all pass false...what do you think about removing the 3rd arg (forcing anyone that wants to control that behavior into the 4-arg flavor that also takes queryParameters)?

Reply:
And actually the 4-arg flavor only has to callers... our JAX-RS Read resource and our FHIRRestInteractionVisitorPersist (called from bundle processing and set to true whether the bundle entry is a read)

Reply:
OK, now I added a commit to remove the 'throwExcOnNull' param from the simpler doRead version